tests/Aspire.Dashboard.Tests/Utils/CertificateUtilTests.cs

```diff
@@ -0,0 +1,132 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.AspNetCore.InternalTesting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Xunit;
+
+namespace Aspire.Dashboard;
+
+public sealed class CertificateUtilTests
+{
+    [Theory]
+    [InlineData("testCert.pfx", X509ContentType.Pkcs12)]
+    [InlineData("https-dsa.pem", X509ContentType.Cert)]
+    public void TestCertificateTypes(string fileName, X509ContentType contentType)
+    {
+        var filePath = TestCertificateLoader.GetCertPath(fileName);
```

Comment:
nit: add arrange, assert, act comments to tests

Reply:
I don't love those comments. Most tests in this repo don't use them.